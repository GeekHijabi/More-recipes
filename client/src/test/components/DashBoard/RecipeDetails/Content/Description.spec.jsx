import React from 'react';
import { shallow } from 'enzyme';
import { Description }
  from '../../../../../js/components/DashBoard/RecipeDetails/Content/Descriptions';

const props = {
  description: 'boil rice',
  activeTab: {
    name: 'description',
  }
};


describe('Description snapshot', () => {
  it('should render correctly', () => {
    const description = shallow(<Description {...props} />);
    expect(description).toMatchSnapshot();
  });
});
