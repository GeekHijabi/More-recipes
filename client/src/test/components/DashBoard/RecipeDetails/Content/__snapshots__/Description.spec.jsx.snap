// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description snapshot should render correctly 1`] = `
<div
  aria-labelledby="directions-tab"
  className="steps tab-pane fade"
  id="directions"
  role="tabpanel"
>
  <div
    className="step"
  >
    <h5>
      Steps
    </h5>
    <p
      className="step_details"
    >
      boil rice
    </p>
  </div>
</div>
`;
